examples/all-clusters-app/all-clusters-common/gen/attribute-size.cpp

```diff
@@ -176,9 +226,16 @@ uint16_t emberAfCopyList(ClusterId clusterId, EmberAfAttributeMetadata * am, boo
             copyListMember(write ? dest : (uint8_t *) &entry->VendorId, write ? (uint8_t *) &entry->VendorId : src, write,
                            &entryOffset, sizeof(entry->VendorId)); // INT16U
             copyListMember(write ? dest : (uint8_t *) &entry->GroupKeyIndex, write ? (uint8_t *) &entry->GroupKeyIndex : src, write,
-                           &entryOffset, sizeof(entry->GroupKeyIndex)); // INT16U
-            copyListMember(write ? dest : (uint8_t *) &entry->GroupKeyRoot, write ? (uint8_t *) &entry->GroupKeyRoot : src, write,
-                           &entryOffset, 16); // OCTET_STRING
+                           &entryOffset, sizeof(entry->GroupKeyIndex));                                   // INT16U
+            chip::ByteSpan * GroupKeyRootSpan = reinterpret_cast<chip::ByteSpan *>(&entry->GroupKeyRoot); // OCTET_STRING
+            if (write && 18 < (kByteSpanInListLengthSize + GroupKeyRootSpan->size()))
```

Comment:
Are these auto-generated? wondering if we need constants for the 31 and 18 values and such.